tailwind.config.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
       'dark-turquoise': '#05b5da',
       'dark-slate-blue': '#3c5c99',
       'trans-blue': '#3e58e1',
+      'trans-blue-action': '#5067e4',
```

Comment:
Hmmm.. isn't "trans" blue `#55CDFC`? 😅 

https://www.schemecolor.com/transgender-pride-flag-colors.php

Reply:
heh, our color looks closer to the brand color of this freight company: https://transblue.org

killer domain name!